libvcx/src/utils/timeout.rs

```diff
@@ -20,4 +24,6 @@ impl TimeoutUtils {
     pub fn some_medium() -> Option<Duration> { Some(TimeoutUtils::medium_timeout()) }
 
     pub fn some_short() -> Option<Duration> { Some(TimeoutUtils::short_timeout()) }
+
+    pub fn some_custom(secs: u64) -> Option<Duration> { Some(TimeoutUtils::custom_timeout(secs)) }
```

Comment:
I wonder why prewrapping into `Some(...)` is necessary (besides it is making the name quite confusing)

Reply:
Good point ... It's usually called like this
```
cb.receive(TimeoutUtils::some_custom(1)).unwrap();
```
this will require small changed in many files across the codebase, we can do that refactor in separate PR